assets/src/content-converter/index.js

```diff
@@ -18,20 +22,37 @@ class ContentConverter extends Component {
 
 		this.state = {
 			isActive: null,
+			retryFailedConversions: props.retryFailedConversions,
 			postIds: '',
 			thisBatch: '',
 			maxBatch: '',
+			hasIncompleteConversions: '',
 		};
 	}
 
 	componentDidMount() {
-		return fetchConversionBatch()
+		const { retryFailedConversions } = this.state;
+
+		// Get a batch of regular conversions, or retry the failed ones.
+		const fetchBatchPromise = ! retryFailedConversions
+			? fetchConversionBatch
+			: fetchRetryFailedConversionsBatch;
+
+		return fetchBatchPromise()
 			.then( response => {
-				if ( response && response.ids ) {
-					const { ids: postIds, thisBatch, maxBatch } = response;
-					this.setState( { postIds, thisBatch, maxBatch, isActive: true } );
-					console.log( ' ----------------------- ABOUT TO CONVERT IDS: ' + postIds );
-					return runMultiplePosts( postIds );
+				if ( response ) {
+					const { ids: postIds, thisBatch, maxBatch, hasIncompleteConversions } = response;
+					this.setState( {
+						postIds,
+						thisBatch,
+						maxBatch,
+						hasIncompleteConversions,
+						isActive: true,
+					} );
+					if ( this.state.postIds ) {
```

Comment:
Thanks so much for sharing this!
Instead of running this conditional in the callback, I changed the conditional to check the Reponse param instead, because it turns out simpler.